src/app/styles/switchback-v1.css

```
@@ -674,21 +674,22 @@
-  background: color-mix(in srgb, var(--machined) 97%, transparent);
+  --oled: var(--sb-ink);
+  --machined: var(--sb-surface);
+  --raised: var(--sb-surface-raised);
+  --instrument: var(--sb-text);
+  --metal: var(--sb-text-muted);
+  --line: var(--sb-border);
+  --ride-forest: var(--sb-text);
+  --ride-ink: var(--sb-text);
+  --ride-cream: var(--sb-surface);
+  --ride-line: var(--sb-border);
+  border-color: var(--sb-border);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)